### PR TITLE
Make logo adapt to dark mode; add source code links to documentation

### PR DESCRIPTION
**Pull request by gamatos**
- Make logo adapt to dark mode (it currently partly blends into the dark background)
- Add source code links to the API in the documentation. These links take the user directly to the range of lines where the function is defined in the source file in the GitHub repository.

### CODE REVIEW

**Comment by gamatos** [comment]
Documentation generated by the CI action is all good ✅

**Comment by gamatos** [comment]
Documentation generated by the CI action is all good ✅

File: docs/conf.py [hunk 1]
```diff
@@ -62,3 +66,24 @@
     "collapse_navigation": False,
     "prev_next_buttons_location": "None",
 }
+
+
+def linkcode_resolve(domain, info):
```
Comment (cqc-melf) [inline docs/conf.py:71]: Can you say a few words for what this function is used for?

Comment (gamatos) [reply docs/conf.py:71]: Great point! I'll add a comment explaining what it does

Comment (gamatos) [reply docs/conf.py:71]: Done!

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**